migrator/src/test/java/com/google/phonenumbers/migrator/MigrationUtilsTest.java

```diff
@@ -38,11 +41,12 @@
 public class MigrationUtilsTest {
 
   private static final String TEST_DATA_PATH = "./src/test/java/com/google/phonenumbers/migrator/testing/testData/";
+  private static final Path RECIPES_PATH = Paths.get(TEST_DATA_PATH + "testRecipesFile.csv");
 
   @Test
   public void getCountryMigratableNumbers_expectNoMatches() throws IOException {
-    String recipesPath = TEST_DATA_PATH + "testRecipesFile.csv";
-    MigrationJob job = MigrationFactory.createCustomRecipeMigration("34", "44", Paths.get(recipesPath));
+    MigrationJob job = MigrationFactory.createCustomRecipeMigration("34", "44", MigrationFactory
```

Comment:
It might be good to create these as a local variable just so we can see what they mean.

Reply:
+1 to making those into a "named something" (either local variable or constant since I believe you're using some of them in multiple tests?)